tests/test_hub.py

```diff
@@ -116,8 +116,15 @@ def _export_onnx(hub, half: bool = False, hold: bool = True):
     return result
 
 
-def _export_waffle(hub):
-    result: ExportWaffleResult = hub.export_waffle()
+def _export_waffle(hub, hold: bool = True):
+    result: ExportWaffleResult = hub.export_waffle(hold=hold)
+
+    if not hold:
+        assert hasattr(result, "callback")
+        while not result.callback.is_finished() and not result.callback.is_failed():
+            time.sleep(1)
```

Comment:
Is there a reason need a sleep(1) ?

Reply:
When the thread runs as background with the hold option False, we wait for the function to end and test it.
At this point, `sleep(1)` was used to prevent excessive use of cpu resources within the loop of the test code.